websites/K/Komik Indo/presence.ts

```diff
@@ -0,0 +1,127 @@
+const presence = new Presence({
+		clientId: "1272521587912081428",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+const enum Assets {
+	Logo = "https://i.imgur.com/HccQ60t.png",
+}
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			details: "Browsing...",
+			largeImageKey: Assets.Logo,
+			startTimestamp: browsingTimestamp,
+		},
+		{ pathname, href, hostname } = document.location,
+		buttons = await presence.getSetting<boolean>("buttons");
+	switch (true) {
+		case hostname === "komikindo.tv":
```

Comment:
This condition is always true regardless of path. Perhaps revert it to the `href` check or similar.